README.md

```diff
@@ -3,6 +3,16 @@ esri-dump
 
 Scrapes an Esri REST endpoint and writes a GeoJSON file.
 
+## Installation
```

Comment:
A note to people unfamiliar with python development that `git clone` isn't needed would be helpful since the last step (I think) does that.